source/extensions/transport_sockets/tls/openssl_impl.h

```diff
@@ -19,7 +19,13 @@ namespace Tls {
 
 int set_strict_cipher_list(SSL_CTX* ctx, const char* str);
 
-STACK_OF(X509) * SSL_get_peer_full_cert_chain(const SSL* ssl);
+// SSL_get_peer_full_cert_chain exists in the BoringSSL library. This diverges in that
+// the caller will own the returned certificate chain and needs to call sk_X509_free on any
```

Comment:
Whoops it seems mentioning `sk_X509_free()` here is inaccurate, that ought to be `sk_X509_pop_free()`

Reply:
Addressed in c3b2e8e5d6926d33cd31341ebf82c47670841159